packages/rancher-alerting-drivers/charts/templates/network-policy-allow-all.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: networking.k8s.io/v1
```

Comment:
Instead of adding the network policy, serviceaccount, psp, and roles manually in these files, let's just copy what we have for Monitoring today here:

https://github.com/rancher/charts/blob/dev-v2.5-source/packages/rancher-monitoring/generated-changes/overlay/templates/rancher-monitoring/hardened.yaml

That way we can standardize this across feature charts, similar to what we do today for helpers.tpl functions.

Reply:
You still need the serviceaccount, rolebinding, and PSP to allow your deployments to be deployed in a hardened cluster, but this ^^ takes care of the scan requirements.

Reply:
changes are made based on the above comments.